iop/system/iomanx/src/iomanX.c

```diff
@@ -1052,6 +1052,7 @@ static const char *parsefile(const char *path, iomanX_iop_device_t **p_device, i
 	if ( devname_len > (sizeof(canon) - 1) )
 		return NULL;
 	strncpy(canon, path_trimmed, devname_len);
+	canon[devname_len] = 0;
```

Comment:
Is this the same than `{0}`

Reply:
No. `{0}` is only used for initializer of an array, but we are setting an element of an array (terminating the string after  `strncpy`)